internal/handlers/dto/user_body.go

```diff
@@ -26,7 +24,8 @@ type UserInformationRequestBody struct {
 	Lastname        string                `json:"lastname"`
 	NationalID      string                `json:"nationalID"`
 	Gender          string                `json:"gender"`
-	BirthDate       time.Time             `json:"birthDate"`
+	BirthDate       string                `json:"birthDate"`
```

Comment:
no need to change to string cuz frontend will send the time string in ISO time string format